.github/workflows/shared_deploy_cloudhub_1_0.yml

```
@@ -44,10 +44,11 @@ on:
         type: string
         required: false
         default: 'OFF'
-      main_api_interface_path:
-        description: Main APIKit interface path
-        type: string
+      disable_log4j2_file_appender:
+        description: Disable Log4j2 File Appender
+        type: boolean
         required: false
+        default: false
```

IMO, using a positive flag could be clearer:
```suggestion
      enable_log4j2_file_appender:
        description: Disable Log4j2 File Appender
        type: boolean
        required: false
        default: true
```
But this is just my opinion 🙏

---

I'm not sure too haha, for me it's the same as we intent to `disable this log appender` on CloudHub. Let's see any other opinion from the team cc @toby-thanathip @sanG-github @vnntsu